### PR TITLE
Fix lints as needed in .pylintrc

### PR DESCRIPTION
**Pull request by ZENALC**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -1,67 +1,33 @@
 [FORMAT]
 max-line-length = 120
+extension-pkg-whitelist=PyQt5
+
+[DESIGN]
+max-attributes=40  # TODO: refactor classes as needed to have default at 7
```
Comment (inverse) [inline .pylintrc:6]: Nice work arounds for now :)

**Review by inverse — APPROVED** [review]
Seems legit 👍🏼

File: .pylintrc [hunk 2]
```diff
@@ -1,67 +1,33 @@
 [FORMAT]
 max-line-length = 120
+extension-pkg-whitelist=PyQt5
```
[the remark this answers is not in the document]
Comment (ZENALC) [reply .pylintrc:3]: No idea. What uic warning?

File: .pylintrc (hunk 1 above)
Comment (ZENALC) [reply .pylintrc:6]: Yep, eventually we will have to refactor though :D

File: algobot/data.py [hunk 3]
```diff
@@ -259,6 +259,7 @@ def get_latest_timestamp(self) -> int:
         """
         result = self.get_latest_database_row()
         if result is None:
+            # pylint: disable=protected-access
```
[the remark this answers is not in the document]
Comment (ZENALC) [reply algobot/data.py:262]: No idea. :P

File: algobot/interface/configuration.py [hunk 4]
```diff
@@ -19,10 +19,10 @@
     add_start_end_step_to_layout, get_default_widget, set_value)
 # noinspection PyUnresolvedReferences
 from algobot.interface.utils import get_elements_from_combobox
-from algobot.strategies import *  # noqa: F403, F401
+from algobot.strategies import *  # noqa: F403, F401 pylint: disable=wildcard-import
```
[the remark this answers is not in the document]
Comment (ZENALC) [reply algobot/interface/configuration.py:22]: Well, this allows all strategies from the strategies class to be imported into the namespace dynamically.

**Review by ZENALC — COMMENTED** [review]
ok

**Pull request merged**